webrtc.html

```diff
@@ -7810,6 +7810,11 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
               <p>When the <code><a>insertDTMF()</a></code> method is invoked,
               the user agent MUST run the following steps:</p>
               <ol>
+                <li>If the <code>tones</code> argument contains any
+                <a>unrecognized</a> characters, throw an
```

Comment:
We refer to arguments by both names and ordinality in this section now. Should we rephrase?